src/main/java/bio/overture/ego/utils/IgnoreCaseSortDecorator.java

```diff
@@ -0,0 +1,71 @@
+package bio.overture.ego.utils;
+
+import static java.util.stream.Collectors.toUnmodifiableList;
+import static org.springframework.util.StringUtils.isEmpty;
+
+import java.util.Map;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+@RequiredArgsConstructor
+public class IgnoreCaseSortDecorator implements Pageable {
+
+  @NonNull private final Pageable delegate;
+
+  /** Decorated methods */
+  @Override
+  public Sort getSort() {
+    val orders =
+        delegate.getSort().stream()
+            .map(o -> new Sort.Order(o.getDirection(), getSortField(o.getProperty())).ignoreCase())
+            .collect(toUnmodifiableList());
+
+    return Sort.by(orders);
+  }
+
+  private String getSortField(String sort) {
```

Comment:
I wasn't sure about the best way to make this generic...i.e. right now this would break sorting on `name` on other entities